pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -2608,6 +2740,7 @@ def StorageSiteBalanceRule(model, s, t):
                 - sum(
                     model.v_F_Trucked[s, k, t] for k in model.s_K if model.p_SKT[s, k]
                 )
+                - model.v_F_StorageEvaporationStream[s, t]
```

Comment:
do we need to add some sort of logic ... evaporation should only occur if volume in pit is non zero -> create an isssue